services/web/server/src/simcore_service_webserver/director_v2.py

```diff
@@ -48,7 +48,7 @@ async def _request_director_v2(
         async with session.request(
             method, url, headers=headers, json=data, **kwargs
         ) as resp:
-            if resp.status >= 400:
+            if resp.status != expected_status.status_code:
```

Comment:
FYI: There could be multiple expected resposes status, it should rather be a tuple? Consider as well ``resp.raise_for_status()`` which should raise an exception that can be handled by the caller

CHANGE: on a more general note, I think it is time to split ``director_v2.py`` into submodules. I suggest
- ``director_v2.py``: setup function and main module-app entrypoint
- ``director_v2_settings.py``: config/settings of the module-app
- ``director_v2_api_client.py``:  helpers to do requests to the director_v2 service (e.g. similar to db repos)
- ``director_v2_handlers.py``: function handlers for the webserver api
This cleanup can perfectly be done in a separate PR if you like.


NOTE: i think we need to come with base client concept (both in aiohttp and httpx) to use everywhere (

Reply:
well I guess it could be a Tuple or List or Set, but for now I needed only one.
just had a quick look at raise_for_status, this will raise for anything above 4XX. 
I agree with the splitting. will do that in a next iteration. And also agree with the base client concept.

Reply:
> just had a quick look at raise_for_status, this will raise for anything above 4XX.

isn't that sufficient for this case?